### PR TITLE
remove-deprecated-nb11

### PR DESCRIPTION
**Pull request by cygnus-x-1**
### Description of the Change

Removed all deprecation warnings except for Object.finalize (used by MemoryManager, would require a rethink of how that tool works) and BooleanStateAction (replaced by the largely undocumented Actions API... maybe another time).

### Alternate Designs

N/A

### Why Should This Be In Core?

N/A

### Benefits

There will be less warning on compile, and less chance of things breaking in the future.

### Possible Drawbacks

N/A

### Verification Process

N/A

### Applicable Issues

N/A

### CODE REVIEW

File: CoreInteractiveGraph/test/unit/src/au/gov/asd/tac/constellation/graph/interaction/GLVisualProcessorGraphTester.java [hunk 1]
```diff
@@ -125,22 +125,22 @@ public static StoreGraph createGraph() {
...
-        Graph g = new DualGraph(graph, false);
-        GraphVisualAccess access2 = new GraphVisualAccess(g);
-        GLVisualProcessor processor = new GLVisualProcessor();
-        final VisualManager visualManager2 = new VisualManager(access2, processor);
-        processor.startVisualising(visualManager2);
-        demo.runDemo(processor, visualManager2);
-        GraphChangeListener gct = (event) -> visualManager2.updateFromIndigenousChanges();
+        final GLVisualProcessorDemo demo = new GLVisualProcessorDemo();
+        final StoreGraph graph = createGraph();
+        final Graph dualGraph = new DualGraph(graph, false);
```
Comment (arcturus2) [inline CoreInteractiveGraph/test/unit/src/au/gov/asd/tac/constellation/graph/interaction/GLVisualProcessorGraphTester.java]: 👍

File: CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/logging/ConstellationLoggerHelper.java [hunk 2]
```diff
@@ -294,22 +296,21 @@ private static void addFileStatistics(final Properties properties, final Collect
...
-        md5Digest.reset();
-
-        byte[] buffer = new byte[1024];
-
+    private static String createHash(final File file) throws IOException, NoSuchAlgorithmException {
         if (!file.exists()) {
             return null;
         }
+        
+        final MessageDigest md5Digest = MessageDigest.getInstance("SHA256");
```
Comment (arcturus2) [inline CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/logging/ConstellationLoggerHelper.java]: `sha256Digest`

**Comment by cygnus-x-1** [comment]
> Love this change @cygnus-x-1. I only have a very minor feedback with a variable name.

Fixed. @algol60 is going to take a look at the two deprecation warnings I couldn't figure out... let's wait for that before merging.

**Comment by cygnus-x-1** [comment]
@arcturus2  - this is ready to be merged now.

**Pull request merged**